src/sdk-reference/kuzzle/sections/constructorDescription_cpp.md

```diff
@@ -0,0 +1,48 @@
+## Arguments
+
+| Argument | Type | Description | Required |
+|--------|------|-------------|------------ |
+| `host` | std::string | The target Kuzzle host  | yes |
+| `options` | kuzzleio::options* | Kuzzle connection configuration | yes |
+
+### host
+
+The Kuzzle host to connect to.  
+Can be a hostname or an IP address.
+
+### options
+
+| Option | Type | Description | Default | Required |
+|---------------|---------|----------------------------------------|---------|---------|
+| `autoQueue` | boolean | Automatically queue all requests during offline mode | `false` | no |
+| `autoReconnect` | boolean | Automatically reconnect after a connection loss | `true` | no |
+| `autoReplay` | boolean | Automatically replay queued requests on a `reconnected` event | `false` | no |
+| `autoResubscribe` | boolean | Automatically renew all subscriptions on a `reconnected` event | `true` | no |
+| `queueTTL` | int | Time a queued request is kept during offline mode, in milliseconds | `120000` | no |
+| `queueMaxSize` | int | Number of maximum requests kept during offline mode | `500` | no |
+| `replayInterval` | Duration | Delay between each replayed requests, in milliseconds | `10` | no |
+| `reconnectionDelay` | Duration | number of milliseconds between reconnection attempts | `1000` | no |
+| `sslConnection` | boolean | Switch Kuzzle connection to SSL mode | `false` | no |
+| `volatile` | VolatileData | Common volatile data, will be sent to all future requests | - | no |
+
+## Getter & Setter
+
+The properties can be writable.  
+For example the property `volatile` you can read it with `getVolatile()` and write it with `setVolatile()`
```

Comment:
"For example, you can read the `volatile` property via `getVolatile()` and set it via `setVolatile()`"